test/functional/test_framework/util.py

```diff
@@ -445,6 +446,14 @@ def set_node_times(nodes, t):
         node.setmocktime(t)
 
 
+def check_node_connections(*, node, num_in, num_out):
+    info = node.getnetworkinfo()
+    if info["connections_in"] != num_in:
+        raise AssertionError("connections_in: %s != %s" % (str(info["connections_in"]), str(num_in)))
+    if info["connections_out"] != num_out:
+        raise AssertionError("connections_out: %s != %s" % (str(info["connections_out"]), str(num_out)))
```

Comment:
why is this changed?

Reply:
To get more friendly messages.

Reply:
The difference is minimal, example: `AssertionError: connections_in: 5 != 3`, with `assert_equal`: `AssertionError: not(5 == 4)`. I can change back to assert_equal if you think its better, np.

Reply:
Yes, considering the stack trace, there is no difference. Going to change back to assert_equal.

Reply:
Done.